geps/gep-713.md

```diff
@@ -1,4 +1,4 @@
-# GEP-713: Metaresources and Policy Attachment_metaresource_
+# GEP-713: Metaresources and Policy Attachment
```

Comment:
A general note, this GEP is _massive_. I think the overall size of it makes an already complicated topic even more overwhelming to understand and/or contribute to. One potential way to mitigate this problem would be to have a separate page that collects and documents problems with policy attachment and links out to potential solutions and how we're trying to improve things here. For example, we could keep something like https://github.com/kubernetes-sigs/gateway-api/discussions/2012 as an interactive forum for proposing solutions/improvements that would eventually turn into GEPs of their own, but the problem statements could be clearly stated somewhere in our docs... 

This is not blocking feedback, I just think we're going to need find some ways to either split this GEP up and/or cut some of the content out so this can be a bit more manageable + approachable.

Reply:
I've logged #2307, and assigned it to myself.